examples/nas/search_space_zoo/nasbench201.py

```diff
@@ -0,0 +1,206 @@
+            data, target = data.to(device), target.to(device)
+            optimizer.zero_grad()
+            output = model(data)
+            loss = criterion(output, target)
+            loss.backward()
+            optimizer.step()
+
+            if batch_idx % args.log_frequency == 0:
+                logger.info('Train Epoch: {} [{}/{} ({:.0f}%)]\tLoss: {:.6f}'.format(
+                            epoch, batch_idx * len(data), len(train_dataloader.dataset),
+                            100. * batch_idx / len(train_dataloader), loss.item()))
+        model.eval()
+        correct = 0
+        test_loss = 0.0
+        for data, target in valid_dataloader:
+            data, target = data.to(device), target.to(device)
+            output = model(data)
+            test_loss += criterion(output, target, reduction='sum').item()
+            pred = output.argmax(dim=1, keepdim=True)
+            correct += pred.eq(target.view_as(pred)).sum().item()
+        test_loss /= len(valid_dataloader.dataset)
+        accuracy = 100. * correct / len(valid_dataloader.dataset)
+        logger.info('\nTest set: Average loss: {:.4f}, Accuracy: {}/{} ({:.0f}%)\n'.format(test_loss, correct,
+                                                                                           len(valid_dataloader.dataset), accuracy))
+        model.train()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser("nb201")
+    parser.add_argument('--stem_out_channels', default=16, type=int)
+    parser.add_argument('--unrolled', default=False, action='store_true')
+    parser.add_argument('--batch_size', default=64, type=int)
+    parser.add_argument('--epochs', default=50, type=int)
+    parser.add_argument('--num_modules_per_stack', default=5, type=int)
+    parser.add_argument('--log-frequency', default=10, type=int)
+    parser.add_argument('--bn_momentum', default=0.1, type=int)
+    parser.add_argument('--bn_affine', default=True, type=bool)
+    parser.add_argument('--bn_track_running_stats', default=True, type=bool)
+    parser.add_argument('--arch', default=None, help='json file which should meet requirements in NAS-Bench-201')
+    parser.add_argument('--visualization', default=False, action='store_true')
+    args = parser.parse_args()
+
+    dataset_train, dataset_valid = datasets.get_dataset("cifar10")
+    model = NASBench201Network(stem_out_channels=args.stem_out_channels,
+                               num_modules_per_stack=args.num_modules_per_stack,
+                               bn_affine=args.bn_affine,
+                               bn_momentum=args.bn_momentum,
+                               bn_track_running_stats=args.bn_track_running_stats)
+
+    optim = torch.optim.SGD(model.parameters(), 0.025)
+    lr_scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optim, args.epochs, eta_min=0.001)
+    criterion = nn.CrossEntropyLoss()
+
+    if args.arch is not None: 
```

Comment:
better to print info here to explicitly tell users that this is model retrain

Reply:
Sure, the architecture will be printed (line 182).

Reply:
not print architecture, but tell users this is model retrain